docs/blogs/EnhancedDevPreviewGatewayAPI/GettingStarted.md

```diff
@@ -229,11 +221,7 @@ spec:
 EOF
 httproute.gateway.networking.k8s.io/http created
 ```
-Wait for the gateway deployment to be ready:
-```console
-$ oc wait -n openshift-ingress --for=condition=ready gateways.gateway.networking.k8s.io demo-gateway
```

Comment:
Would it make sense to keep this but change "Ready" to "Programmed"?

Reply:
+1

Reply:
Was this change inadvertantly deleted?